assets/styles/_variables.scss

```diff
@@ -4,16 +4,24 @@ $breakpoint-tablet-horizontal: 1024px;
 $breakpoint-desktop: 1280px;
 $breakpoint-desktop-wide: 1440px;
 
-// Sizing variables
+/**
+ * Dynamic CSS Variables are defined in assets/styles/variables.php.
+ * You can see generated variables and their values in the <head> of the site.
+ */
+
 :root {
-  --container-max-width: var(--container-max-width);
-  --container-padding: var(--container-padding-mobile);
-  --content-max-width: var(--content-max-width);
-  --content-max-width-large: var(--content-max-width-large);
+  --container-max-width: 1440px;
+  --container-padding-base: 16px;
+  --content-max-width: 690px;
+  --content-max-width-large: 860px;
+  --gutter-width: 24px;
+
+  --container-padding: var(--container-padding-base);
   --component-spacing: var(--component-spacing-base);
-  --gutter-width: var(--gutter-width);
-  --box-spacing: var(--box-spacing);
-  --box-border-radius: var(--box-border-radius);
+
+  --container-padding-mobile: var(--container-padding-base);
+  --container-padding-tablet: calc(var(--container-padding-base) * 2);
+  --container-padding-desktop: calc(var(--container-padding-base) * 4);
 
   --component-spacing-mobile: var(--component-spacing-base);
   --component-spacing-tablet: calc(var(--component-spacing-base) / 3 * 6);
```

Comment:
In this case it can be 2 but I would leave it like this because it shows the formula how this works. Tablet spacing is multiplier of 6 and desktop of 8. Also if you would want to use the `* 5` you can. That would give us 80 and that will still follow our spacing rules.